packages/DatePicker/src/components/ShortcutPanel/ShortcutPanel.js

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+import momentPropTypes from "react-moment-proptypes";
+
+import ArrowLeft from "@paprika/icon/lib/ArrowLeft";
+import ArrowRight from "@paprika/icon/lib/ArrowRight";
+import Button from "@paprika/button";
+import tokens from "@paprika/tokens";
+import useI18n from "@paprika/l10n/lib/useI18n";
+
+import {
+  ActionBarStyled,
+  ColumnHeaderStyled,
+  ContainerStyled,
+  LabelStyled,
+  MonthListStyled,
+  OptionWrapperStyled,
+  PanelContentStyled,
+  YearListStyled,
+} from "./ShortcutPanel.styles";
+
+const propTypes = {
+  date: momentPropTypes.momentObj.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  onConfirm: PropTypes.func.isRequired,
+};
+
+function ShortcutPanel(props) {
+  const { date, onCancel, onConfirm } = props;
+  const [pageIndex, setPageIndex] = React.useState(0);
+  const [selectedMonth, setSelectedMonth] = React.useState(date.month());
+  const [selectedyear, setSelectedYear] = React.useState(date.year());
+  const I18n = useI18n();
+
+  const panelRef = React.useRef(null);
+
+  React.useEffect(() => {
+    panelRef.current.focus();
+  }, [date]);
+
+  function handleChange(event) {
+    const name = event.target.name;
+    const id = event.target.id;
+
+    switch (name) {
+      case "month":
+        setSelectedMonth(id * 1);
+        break;
+      case "year":
+        setSelectedYear(id * 1);
+        break;
+      default:
+        break;
+    }
+  }
+
+  function handleConfirm() {
+    onConfirm({
+      month: selectedMonth,
+      year: selectedyear,
+    });
+  }
+
+  function renderMonthList() {
+    return moment.monthsShort().map((month, monthIndex) => (
+      <OptionWrapperStyled key={month}>
+        <input
+          type="radio"
+          name="month"
+          id={monthIndex}
+          value={month}
+          defaultChecked={monthIndex === selectedMonth}
+          onChange={handleChange}
+        />
+        <LabelStyled htmlFor={monthIndex} isSelected={monthIndex === selectedMonth}>
+          {month}
+        </LabelStyled>
+      </OptionWrapperStyled>
+    ));
+  }
+
+  function renderYearList() {
+    const SELECTED_YEAR_POSITION = 5;
```

Comment:
What does this 5 mean?

Reply:
The selected year's position in shortcut panel
![image](https://user-images.githubusercontent.com/38733362/59884026-d375b080-936b-11e9-9bd1-fe51abcdb65b.png)